wafer/talks/forms.py

```diff
@@ -25,6 +25,24 @@ def __init__(self, *args, **kwargs):
                             _('Delete')))))
         else:
             self.helper.add_input(submit_button)
+        # insert before authors
+        # TODO: this is *YUK*, but if we add the field to Meta.fields below,
+        # then we can disable the widget, sure, but it'd still load all the 
+        # data, and we'd need to validate that the submitted form isn't trying
+        # to change the data if someone bypasses (removes) the HTML disabled
+        # tag. The following is basically display-only as the input field
+        # has no name.
```

Comment:
I've since come up with a better way shown in 8805473d5082aa656e7446eb1e7042cf1fa56080. I didn't do this at this stage as I tried to minimise changes, but if refactoring is an option, I do think we can stay with ModelForm.